app/services/idv/session.rb

```diff
@@ -2,7 +2,7 @@ module Idv
   class Session
     VALID_SESSION_ATTRIBUTES = [
       :resolution, :vendor, :applicant, :params, :profile_id, :recovery_code,
-      :financials_confirmation, :phone_confirmation
+      :financials_confirmation, :phone_confirmation, :profile_confirmation
```

Comment:
Sure, I was just going with what was there to make the changes in this PR as small as possible. I'm happy to alphabetize in a new commit in this PR so you can see just that change.